specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/expressRoutePort.json

```diff
@@ -0,0 +1,763 @@
+                },
+                "links": {
+                    "title" : "ExpressRouteLink Sub-Resources",
+                    "description" : "The set of physical links of the ExpressRoutePort resource",
+                    "readOnly": false,
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ExpressRouteLink"
+                    }
+                },
+                "circuits": {
+                    "readOnly": true,
+                    "type": "array",
+                    "items": {
+                        "$ref": "./network.json#/definitions/SubResource"
+                    },
+                    "description": "Reference the ExpressRoute circuit(s) that are provisioned on this ExpressRoutePort resource."
+                },
+                "provisioningState": {
+                    "type": "string",
+                    "readOnly": true,
+                    "description": "The provisioning state of the ExpressRoutePort resource. Possible values are: 'Succeeded', 'Updating', 'Deleting', and 'Failed'."
+                },
+                "resourceGuid": {
+                    "type": "string",
+                    "description": "The resource GUID property of the ExpressRoutePort resource."
+                }
+            }
+        },
+        "ExpressRoutePort": {
+            "title": "ExpressRoute Port",
+            "description": "ExpressRoutePort resource definition.",
+            "properties": {
+                "properties": {
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/ExpressRoutePortPropertiesFormat",
+                    "description": "ExpressRoutePort properties"
+                },
+                "etag": {
+                    "readOnly": true,
+                    "type": "string",
+                    "description": "A unique read-only string that changes whenever the resource is updated."
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "./network.json#/definitions/Resource"
+                }
+            ]
+        },
+        "ExpressRoutePortListResult": {
+            "title": "ExpressRoute Port List Result",
+            "description": "Response for ListExpressRoutePorts API service call.",
+            "properties": {
+                "value": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ExpressRoutePort"
+                    },
+                    "description": "A list of ExpressRoutePort resources."
+                },
+                "nextLink": {
+                    "type": "string",
+                    "description": "The URL to get the next set of results."
+                }
+            }
+        }
+    },
+    "parameters": {
```

Comment:
Please remove this part and update references to [these items](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/network.json#L149)